### DISCUSSION TITLE
Inquiry About Pyarmor 9.0.5 Compatibility with Python 3.9

### DISCUSSION THREAD

**Discussion by gino2013** [dashingsoft/pyarmor#2022] [Q&A]
Dear Pyarmor Support Team,

I hope this email finds you well. I am currently evaluating Pyarmor 9.0.5 for a project, and I would like to confirm whether this version is fully compatible with Python 3.9.

Could you please clarify if there are any known limitations or specific configurations required for using Pyarmor 9.0.5 with Python 3.9?

Thank you in advance for your assistance, and I look forward to your response.

Best regards,
Lucian

**Comment by jondy** [comment]
Please check this page
https://pyarmor.readthedocs.io/en/latest/topic/obfuscated-script.html#the-differences-of-obfuscated-scripts
